src/Documentation/DocumentationGenerator.php

```diff
@@ -174,6 +185,9 @@ public function generateFixerDocumentation(FixerInterface $fixer)
 
         $doc = "{$titleLine}\n{$title}\n{$titleLine}";
 
+        $srcPath = (new \ReflectionClass($fixer))->getFileName();
+        $doc .= "\n\n`src <{$this->makeRelativePathToDoc($srcPath, 2)}>`_";
```

Comment:
if `src`, them maybe also `tests` ?

Reply:
why not ;-)

what is the best way to get a test path for a given fixer class?

Reply:
reflection->getFileName is good

Reply:
my reasoning is that actually from `tests` we get a lot of examples, from `src` file we don't get that much as users